Symfony/CS/Fixer/Symfony/BlanklineAfterOpenTagFixer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Ceeram <ceeram@cakephp.org>
+ */
+class BlanklineAfterOpenTagFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+        foreach ($tokens as $index => $token) {
```

Comment:
I asked because i would prefer to avoid iterating all tokens from back to front. Currently it breaks out of the loop after the fix of the open tag. Doing it the otehr way around, would mean many unneeded iterations.